package-lock.json

```diff
@@ -192,15 +192,6 @@
         "node": ">=6.9.0"
```

Comment:
If you haven't changed anything in the package.json file, is there a reason the package.lock needs to change? Otherwise it should not be included in this PR because there is always a danger of breaking production dependencies.
A tip: if you want to install dependencies from a branch but don't want to change your package.lock, you can use `npm ci` instead.

Reply:
I did not mean to change these. I think my machine has been evaluating `npm install` differently somehow.